### DISCUSSION TITLE
Unix Login Name RBAC Behavior when Using Multiple Roles and/or Access Requests

### DISCUSSION THREAD

**Discussion by corkrean** [gravitational/teleport#10762] [Show and tell]
Scenario:
  -Role A: Provides access to node x and allows login as "ubuntu"
  -Role B: Provides access to node y and allows login as "root"

If at anytime a user has assumed both roles either via access requests or manual role assignment, they will not be allowed to log in as ubuntu on node y or root on node x.

**Comment by philip-teleport** [comment]
These issues are requesting the same behaviour: [3311](https://github.com/gravitational/teleport/issues/3311) [9896](https://github.com/gravitational/teleport/issues/9896)
